beacon-chain/core/epoch/precompute/attestation_test.go

```diff
@@ -17,7 +18,7 @@ import (
 )
 
 func TestUpdateValidator_Works(t *testing.T) {
-	e := uint64(params.BeaconConfig().FarFutureEpoch)
+	e := types.Slot(params.BeaconConfig().FarFutureEpoch)
```

Comment:
yep, it is a known quirk in our config file -- it is just a huge number essentially, but within our codebase it is used as epoch and as slot. For example:
https://github.com/prysmaticlabs/prysm/blob/7f6b15271a675412d99fbccd4660b702976f13b8/validator/client/validator.go#L308

there it is compared with `types.Epoch` variable.

Since we have it as `FarFutureEPOCH`, I've decided to make it of type `Epoch`.